incubator/kafka/templates/service-brokers-external.yaml

```diff
@@ -8,7 +8,11 @@
     {{- $externalListenerPort := add $root.Values.external.firstListenerPort $i }}
     {{- $responsiblePod := printf "%s-%d" (printf "%s" $fullName) $i }}
     {{- $distinctPrefix := printf "%s-%d" $dnsPrefix $i }}
+    {{- $loadBalancerIP := "" }}
+    {{- if eq $root.Values.external.type "LoadBalancer" }}
```

Comment:
Just because external type is set to LoadBalancer doesn't mean we're using static IPs. I think the chart should support both static and dynamic IPs. I think the `if` statement should check if loadBalancerIP is set or not. Not just if `external.type` is `LoadBalancer`.